### PR TITLE
Server branch starting

### PR DESCRIPTION
**Pull request by cybinon**
PR Checklist

- [ ] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Developer experience (improves developer workflows for contributing to the project)

## Description

_What this PR does_

## Git Issues

Closes #

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [`development` channel](https://discord.gg/REUQRw469e).

### CODE REVIEW

File: packages/config/utils.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        proxy_set_header X-Real-IP $remote_addr;
+    }
+    location /api/ {
+        proxy_pass http://${branchName}:4000/;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+    }
+}`
+
+  fs.writeFileSync(`./nginx/${branchName}.${process.env.DOMAIN_SERVER}.conf`, nginxConf)
```
[the remark this answers is not in the document]
Comment (cybinon) [reply packages/config/utils.js:45]: Sanitized

File: packages/config/utils.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+const removeNginxFile = (branchName) => {
+  if (fs.existsSync(`./nginx/${branchName}.${process.env.DOMAIN_SERVER}.conf`)) fs.unlinkSync(`./nginx/${branchName}.${process.env.DOMAIN_SERVER}.conf`)
```
[the remark this answers is not in the document]
Comment (cybinon) [reply packages/config/utils.js:50]: Sanitized

[the remark this answers is not in the document]
Comment (cybinon) [reply packages/config/utils.js:50]: Sanitized

**Review by KhenBai — APPROVED** [review]
looks solid

**Pull request merged**